apps/settings/locales/settings.en-US.properties

```diff
@@ -955,7 +955,7 @@ clear=Clear
 
 # Security :: Privacy Panel
 privacy-controls=Privacy Controls
-no-privacy-panel=No privacy panel app found
+no-privacy-panel=No privacy controls app found
```

Comment:
please change the naming from `no-privacy-panel` to `no-privacy-controls`, so the l10n community can detect the keyword change and made new translation